File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;

        try {
            int i = Integer.parseInt(s);
        } catch (NumberFormatException nfe) {
            System.out.println(s);
            return false;

        }

        if (s.length() != 4 && s.length() != 6){
            return false;
        }

        return answer;
    }
}

File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;


    for (int i = 0 ; i < s.size() ; i++){
        if (isdigit(s[i]) == 0){
            answer = false;
        } 
    }

    if (s.size() != 4 && s.size() != 6){
        answer = false;
    }


    return answer;
}